---
abstract: Identifying genomic regions influenced by natural selection provides
  fundamental insights into the genetic basis of local adaptation. However, it
  remains challenging to detect loci under complex spatially varying selection.
  We propose a deep learning-based framework, DeepGenomeScan, which can detect
  signatures of spatially varying selection. We demonstrate that DeepGenomeScan
  outperformed principal component analysis- and redundancy analysis-based
  genome scans in identifying loci underlying quantitative traits subject to
  complex spatial patterns of selection. Noticeably, DeepGenomeScan increases
  statistical power by up to 47.25% under nonlinear environmental selection
  patterns. We applied DeepGenomeScan to a European human genetic dataset and
  identified some well-known genes under selection and a substantial number of
  clinically important genes that were not identified by SPA, iHS, Fst and
  Bayenv when applied to the same dataset.
slides: null
url_pdf: https://academic.oup.com/bib/article-pdf/23/5/bbac354/45936671/bbac354.pdf
publication_types:
  - "2"
authors:
  - admin
  - Charleston W K Chiang
summary: Deciphering signatures of natural selection via deep learning.
url_dataset: ""
url_project: ""
author_notes: []
publication_short: Brief. Bioinformatics.
url_source: ""
url_video: ""
publication: "*Briefings in Bioinformatics, 23*(5)"
featured: false
date: 2022-09-02T00:00:00.000Z
url_slides: ""
title: Deciphering signatures of natural selection via deep learning
tags:
  - Source Themes
projects: []
image:
  caption: "Image credit: [**Unsplash**](https://unsplash.com/photos/jdD8gXaTZsc)"
  focal_point: ""
  preview_only: false
publishDate: 2023-10-12T00:00:00.000Z
url_poster: ""
url_code: ""
doi: 10.1111/1755-0998.13606
---
