---
title: Deciphering signatures of natural selection via deep learning
publication_types:
  - "2"
authors:
  - admin
  - Charleston WK Chiang
  - Oscar E Gaggiotti
doi: 10.1093/bib/bbac354
publication: Briefings in Bioinformatics
publication_short: Brief. Bioinformatics.
abstract: Geographic patterns of human genetic variation provide important
  insights into human evolution and disease. A commonly used tool to detect and
  describe them is principal component analysis (PCA) or the supervised linear
  discriminant analysis of principal components (DAPC). However, genetic
  features produced from both approaches could fail to correctly characterize
  population structure for complex scenarios involving admixture. In this study,
  we introduce Kernel Local Fisher Discriminant Analysis of Principal Components
  (KLFDAPC), a supervised non-linear approach for inferring individual
  geographic genetic structure that could rectify the limitations of these
  approaches by preserving the multimodal space of samples. We tested the power
  of KLFDAPC to infer population structure and to predict individual geographic
  origin using neural networks. Simulation results showed that KLFDAPC has
  higher discriminatory power than PCA and DAPC. The application of our method
  to empirical European and East Asian genome-wide genetic datasets indicated
  that the first two reduced features of KLFDAPC correctly recapitulated the
  geography of individuals and significantly improved the accuracy of predicting
  individual geographic origin when compared to PCA and DAPC. Therefore, KLFDAPC
  can be useful for geographic ancestry inference, design of genome scans and
  correction for spatial stratification in GWAS that link genes to adaptation or
  disease susceptibility.
draft: false
featured: false
image:
  filename: ai_genomics.jpg
  focal_point: Smart
  preview_only: false
date: 2023-10-13T18:23:58.005Z
---
